src/api/index.js

```diff
@@ -305,10 +305,16 @@ export function getCondition({ name, namespace }) {
   return get(uri);
 }
 
-export function getPodLogURL({ container, name, namespace }) {
+export function getPodLogURL({ container, name, namespace, follow }) {
   let queryParams;
-  if (container) {
-    queryParams = { container };
+  if (container || follow) {
+    queryParams = {};
+    if (container) {
+      queryParams.container = container;
+    }
+    if (follow) {
+      queryParams.follow = follow;
+    }
   }
```

Comment:
This can be condensed as:

```js
const queryParams = {
  ...(container && { container }),
  ...(follow && { follow })
};
```

Empty object for queryParams should be treated the same as null / undefined so it's safe to do this regardless of the values of `container` or `follow`, and it'll only include the keys with a truthy value.